### PR TITLE
Add example for using Vault to get the Audit token.

### PR DESCRIPTION
**Pull request by govind-salinas-pangea**

### CODE REVIEW

File: examples/vault/get.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+	auditTokenID := os.Getenv("PANGEA_AUDIT_TOKEN_ID")
+	if auditTokenID == "" {
+		log.Fatal("Unauthorized: No audit token id present")
+	}
+    vaultConfig := pangea.Config{
+		Token:  token,
+		Domain: os.Getenv("PANGEA_DOMAIN"),
+	}
+	vaultObj := vault.New(&vaultConfig)
```
Comment (pangea-andrest) [inline examples/vault/get.go:29]: We usually call it `client` instead of object

File: examples/vault/get.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+	fmt.Println("Fetch the audit token...")
+	getRequest := &vault.GetRequest{
+        ID: auditTokenID,
+	}
+	storeResponse, err := vaultObj.Get(ctx, getRequest)
+	if err != nil {
+		log.Fatal(err)
+	}
+    auditToken := storeResponse.Result.CurrentVersion.Secret
```
Comment (pangea-andrest) [inline examples/vault/get.go:41]: image
Is indentation wrong or it's just github when displaying it?

File: examples/vault/get.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+	fmt.Println("Initialize Log...")
+    auditConfig := pangea.Config{
+		Token:  *auditToken,
```
Comment (pangea-andrest) [inline examples/vault/get.go:45]: Dereferenced without a nil check

**Review by pangea-andrest — CHANGES_REQUESTED** [review]

File: examples/vault/get.go (hunk 1 above)
Comment (govind-salinas-pangea) [reply examples/vault/get.go:29]: it was called *cli before which normally stands for command line which i found confusing, i dont mind calling it "client" tho

File: examples/vault/go.mod [hunk 4]
```diff
@@ -2,7 +2,10 @@ module examples/vault
 
 go 1.19
 
-require github.com/pangeacyber/pangea-go/pangea-sdk/v3 v3.5.0
+require (
+	github.com/pangeacyber/pangea-go/pangea-sdk/v2 v2.3.0
```
Comment (pangea-andrest) [inline examples/vault/go.mod:6]: It should use only v3

File: examples/vault/get.go [hunk 5]
```diff
@@ -0,0 +1,65 @@
+// vault rotate is an example of how to use the rotate method
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v2/pangea"
```
Comment (pangea-andrest) [inline examples/vault/get.go:10]: Update to v3

**Review by pangea-andrest — APPROVED** [review]

**Pull request merged**